contracts/market-wrapper/NounsMarketWrapper.sol

```diff
@@ -0,0 +1,133 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.5;
+
+// ============ External Imports ============
+import {INounsAuctionHouse} from "../external/interfaces/INounsAuctionHouse.sol";
+
+// ============ Internal Imports ============
+import {IMarketWrapper} from "./IMarketWrapper.sol";
+
+/**
+ * @title NounsMarketWrapper
+ * @author Anna Carroll + Nounders
+ * @notice MarketWrapper contract implementing IMarketWrapper interface
+ * according to the logic of Nouns' Auction Houses
+ */
+contract NounsMarketWrapper is IMarketWrapper {
+    // ============ Internal Immutables ============
+
+    INounsAuctionHouse internal immutable market;
```

Comment:
i know we make this `internal` for `Zora` but may be handy/useful to make `public`